package com.tsola2002.learnjava.ch04_exceptions;

public class TryCatchFinally {

  public static void main(String[] args) {
    //example3();
    int x = 4;


  }

  private static void example3(){
    int x = 4;
    assert (x > 3);
  }
}
